src/index.js

```diff
@@ -39,20 +37,25 @@ const rootToHostMap = new WeakMap();
 const rootToSlotMap = new WeakMap();
 
 
-// * WebKit only *
-//
-// We require some way to parse HTML natively because we can't use the native
-// accessors.
-
+// Unfortunately manual DOM parsing is because of WebKit.
 const parser = new DOMParser();
 function parse (html) {
   const tree = document.createElement('div');
+
+  // Everything not WebKit can do this easily.
+  if (canPatchNativeAccessors) {
```

Comment:
Turns out we can use the native accessors here instead of manually parsing and importing nodes!